stylesheets/commons/FilterButtons.less

```diff
@@ -58,6 +58,44 @@ Author(s): Elysienna
 	animation-duration: 0.8s;
 }
 
+.dota2-test {
+	.filter-buttons {
+		display: flex;
+		justify-content: start;
+		flex-flow: row wrap;
+		font-size: 0.825rem;
+		margin: 0.125rem;
+		align-items: center;
+	}
+
+	.filter-button {
+		display: flex;
+		align-items: center;
+		border-radius: 0.5rem;
+		padding: 0.375rem 0.75rem;
+		font-weight: bold;
+		flex-grow: initial;
+
+		.theme--light & {
+			color: var( --clr-on-primary-container, #ffffff );
+			background-color: var( --clr-primary-container, #1f3d7a );
+			border-color: var( --clr-primary-container, #1f3d7a );
+		}
+
+		.theme--dark & {
+			color: #ffffff;
+			background-color: var( --clr-surface-3, #282828 );
+			border-color: var( --clr-surface-3, #282828 );
+		}
+
+		&.filter-button--active {
+			background-color: var( --clr-wiki-theme-primary, #1f3d7a );
+			border-color: var( --clr-wiki-theme-primary, #1f3d7a );
+			color: #ffffff;
+		}
+	}
```

Comment:
Please do test this first (also for bruinen). This is to ensure the default buttons on light mode have the correct background color shade. It's a bit of an unusual way to do it but it's because you cannot edit the alpha value of the variable. That's why I've done it in the after pseudo element.
```suggestion
	.filter-button {
		display: flex;
		align-items: center;
		border-radius: 0.5rem;
		padding: 0 0.75rem;
		font-weight: bold;
		flex-grow: initial;
		position: relative;
		border: 0;
		min-height: 2rem;

		.theme--light & {
			color: var( --clr-wiki-theme-primary, #ffffff );
		}

		.theme--dark & {
			color: #ffffff;
		}
		
		&:not( .is--active )::after {
			content: '';
			opacity: 0.12;
			position: absolute;
			inset: 0;
			z-index: -1;
			border-radius: 0.5rem;

			.theme--light & {
				background-color: var( --clr-wiki-theme-primary, #1f3d7a );
			}

			.theme--dark & {
				background-color: #fff;
			}
		}
		
		@media ( hover: hover ) {
			&:hover {
				&::after {
					opacity: 0.24;
				}
			}
		}

		&.filter-button--active {
			background-color: var( --clr-wiki-theme-primary, #1f3d7a );
			color: #ffffff;
		}
	}
```

Reply:
Updated